osquery/dispatcher/scheduler.cpp

```diff
@@ -21,15 +21,14 @@
 #include "osquery/core/process.h"
 #include "osquery/database/query.h"
 #include "osquery/dispatcher/scheduler.h"
-#include "osquery/sql/sqlite_util.h"
 
 namespace osquery {
 
 FLAG(bool, enable_monitor, false, "Enable the schedule monitor");
 
 FLAG(uint64, schedule_timeout, 0, "Limit the schedule, 0 for no limit")
 
-inline SQL monitor(const std::string& name, const ScheduledQuery& query) {
+SQLInternal monitor(const std::string& name, const ScheduledQuery& query) {
```

Comment:
I removed the `inline` here because there's really no performance improvement. There are tests that use it and I'd rather break them in the compile phase if there's ever method signature differences.